### PR TITLE
Update meta module info in package.json

### PR DESCRIPTION
**Pull request by mrmrs**
@donnieberg @johnotander 

If this looks good feel free to merge.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,5 +33,9 @@
     "build:docs": "tachyons src/tachyons-background-size.css --generate-docs --package=../../package.json > readme.md",
     "build": "npm run build:css && npm run build:minify && npm run build:docs",
     "build:watch": "watch 'npm run build' ./src"
+  },
+  "tachyonsModule": {
+    "category": "theming",
+    "isCore": "true"
```
Comment (johno) [inline package.json]: We can make this a JSON boolean with `"isCore": true`.

**Comment by johno** [comment]
👍  Looks good to me, with or without the minor nitpick change.

**Comment by mrmrs** [comment]
Oh good to know. I will make that change.

**Comment by donnieberg** [comment]
Looks good to me too! Thanks Adam :)

**Comment by donnieberg** [comment]
@mrmrs if you update the string "true" to a bool, I'll go ahead and merge that in. I'm also happy to update some of the other modules' `package.json`. Just lemme know :)

**Comment by mrmrs** [comment]
Updated :)

**Pull request merged**